### PR TITLE
task02

### PR DESCRIPTION
**Pull request by dmialeshka**

### CODE REVIEW

File: tasks/task02/triangle_area.py [hunk 1]
```diff
@@ -0,0 +1,56 @@
+import math
```
Comment (justas-exadel) [inline tasks/task02/triangle_area.py:11]: Please add a task description as a doc string at the top.
"""
Task description. 
"""

File: tasks/task02/triangle_area.py [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+    2. Calculate triangle area by 2 sides and angle between them
+    3. Exit
+    """)
+    number: int = int(input('Enter menu item number: '))
+    operations = {
+        1: base_calculate,
+        2: calculate_with_angle,
+        3: close
+    }
+    operation = operations.get(number)
```
Comment (justas-exadel) [inline tasks/task02/triangle_area.py:57]: What will happen if a user inputs incorrect input value?

File: tasks/task02/triangle_area.py [hunk 3]
```diff
@@ -0,0 +1,56 @@
+import math
+
+
+def get_values(data: str):
+    values = data.split(' ')
+    first_v = int(values[0])
```
Comment (justas-exadel) [inline tasks/task02/triangle_area.py]: It seems this function could be rewritten as 
```
def get_values(data:str):
   return map(int, data.split())
```

File: tasks/task02/triangle_area.py [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+    3. Exit
+    """)
+    number: int = int(input('Enter menu item number: '))
+    operations = {
+        1: base_calculate,
+        2: calculate_with_angle,
+        3: close
+    }
+    operation = operations.get(number)
+    operation()
```
Comment (justas-exadel) [inline tasks/task02/triangle_area.py:60]: If I understand correctly here you want to call `close()`

**Review by justas-exadel — COMMENTED** [review]
Please check the comments.

File: tasks/task02/triangle_area.py (hunk 1 above)
Comment (dmialeshka) [reply tasks/task02/triangle_area.py:11]: Added

File: tasks/task02/triangle_area.py (hunk 2 above)
Comment (dmialeshka) [reply tasks/task02/triangle_area.py:57]: Added fallback

File: tasks/task02/triangle_area.py (hunk 3 above)
Comment (dmialeshka) [reply tasks/task02/triangle_area.py]: Rewrited

File: tasks/task02/triangle_area.py (hunk 4 above)
Comment (dmialeshka) [reply tasks/task02/triangle_area.py:60]: Nope, it's call a function regarding operation 1,2,3

**Pull request merged**